ensime_shared/client.py

```diff
@@ -85,164 +88,144 @@ def setup_logger():
 
         super(EnsimeClient, self).__init__()
         self.editor = editor
-        self.launcher = launcher
+        self.config = config
 
         self.log = setup_logger()
         self.log.debug('__init__: in')
         self.editor.initialize()
 
         self.ws = None
-        self.ensime = None
```

Comment:
Indeed, I was aware but hadn't left a todo comment, thanks for logging it here.

`:EnClients` is currently undocumented, I think it should be renamed to `:EnStatus` to reflect that it's really as much about server statuses, and the output formatting enhanced. This could even become a special filetype window for a "console" in our server lifecycle plans, with some summarized key mappings for performing actions like restarting the server on the current line, etc.

I'll just make some minimal fix here so it isn't utterly broken, until we give `EnsimeProcess` some attention in forthcoming work.